to_do_app/lib/detailTask/detail_task_cubit.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:async';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../addTask/task_state.dart';
+import '../domain/card_to_do.dart';
+import '../repository/repository_task.dart';
+
+class DetailTaskCubit extends Cubit<TaskState> {
+  final repo = RepositoryTask();
+  final CardToDo element;
+  final int index;
+  StreamSubscription<CardToDo>? subscription;
+
+  DetailTaskCubit(this.index)
+      : element = RepositoryTask().getTask(index),
+        super(TaskState(task: RepositoryTask().getTask(index))) {
+    subscription = repo.getTaskStream(index).listen((event) {
+      print(event.toString());
```

Comment:
we usually don't keep prints on the final code unless we want to record a failure or a specific behavior, remove it